ignite/templates/module/create/files/ibc/x/{{moduleName}}/client/cli/tx.go.plush

```diff
@@ -7,24 +7,25 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	"%s/x/%s/types"
+	// "github.com/cosmos/cosmos-sdk/client/flags"
+	"<%= modulePath %>/x/<%= moduleName %>/types"
 )
 
 var DefaultRelativePacketTimeoutTimestamp = uint64((time.Duration(10) * time.Minute).Nanoseconds())
-			
+
 const listSeparator = ","
 
-// GetTxCmd returns the transaction commands for this module
+// GetTxCmd returns the transaction commands for this module.
```

Comment:
It does not look like it gets scaffolded only if it is an IBC module.

Reply:
it is, You can check the template path:

`ignite/templates/module/create/files/ibc/x/{{moduleName}}/client/cli/tx.go.plush`

Reply:
You are very right!